pyproject.toml

```diff
@@ -13,8 +13,8 @@ packages = [{include = "prefab_cloud_python"}, {include = "prefab_pb2.py"}, {inc
 
 [tool.poetry.dependencies]
 python = ">= 3.9, < 4"
-grpcio = "^1.51.3"
-grpcio-tools = "^1.51.3"
+grpcio = ">= 1.48.4"
+grpcio-tools = ">= 1.48.4"
```

Comment:
I think moving this to [tool.poetry.group.dev.dependencies] actually relaxes the protobuf requirements even further and that would be a good thing.